src/battle_setup.c

```diff
@@ -954,9 +963,12 @@ static void CB2_EndTrainerBattle(void)
         }
         else
         {
-            SetMainCallback2(CB2_ReturnToFieldContinueScriptPlayMapMusic);
-            SetBattledTrainerFlag();
-            QuestLogEvents_HandleEndTrainerBattle();
```

Comment:
Are we planning on utilizing the quest log at all, or did you purposefully exclude this line from the new `if` statement?

Reply:
Good catch. I will leave it in as defeating an NPC makes sense to keep track of.